assets/js/modules/analytics/components/common/EnableUniversalAnalytics.js

```diff
@@ -20,25 +20,37 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { useMount } from 'react-use';
 
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { Fragment, useCallback } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	Fragment,
+	useCallback,
+	createInterpolateElement,
+} from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { Switch } from 'googlesitekit-components';
 import { CORE_FORMS } from '../../../../googlesitekit/datastore/forms/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import { MODULES_ANALYTICS, FORM_SETUP } from '../../datastore/constants';
+import { TYPE_INFO } from '../../../../components/SettingsNotice';
 import ProfileSelect from './ProfileSelect';
 import PropertySelect from './PropertySelect';
+import SettingsNotice from '../../../../components/SettingsNotice/SettingsNotice';
+import WarningIcon from '../../../../../../assets/svg/icons/warning-icon.svg';
 const { useSelect, useDispatch } = Data;
 
-export default function EnableUniversalAnalytics( { children } ) {
+export default function EnableUniversalAnalytics( {
+	children,
+	hasModuleAccess = true,
```

Comment:
Using `true` as a default here because we're also re-using this component in setup, and don't need to check module access at this level there.